finetune/adapter.py

```diff
@@ -141,7 +141,7 @@ def train(
     total_lengths = 0
     total_t0 = time.perf_counter()
 
-    for iter_num in range(max_iters):
+    for iter_num in range(1, max_iters + 1):
```

Comment:
I think this is simpler to reason if it starts at 1. For instance, the log_interval `if` block is missing the +1